lib/index.js

```diff
@@ -162,6 +163,14 @@ module.exports.gzip = function(options) {
       initFile(file);
 
       // add content-type header
+      mimeType = mime.lookup(file.path);
```

Comment:
can you refactor that into a new function to avoid duplicating the logic?
alertnatively we could also keep track of the original path like this file.s3.unzip = file.path
and calculate the content-type on file.s3.unzip || file.s3.path